doccano_client/beta/controllers/comment.py

```diff
@@ -48,7 +48,12 @@ def __init__(self, parent_url: str, client_session: Session) -> None:
     @property
     def comments_url(self) -> str:
         """Return an api url for comments list of an object"""
-        return f"{self._parent_url}/comments"
+        if "/examples" in self._parent_url:
```

Comment:
A quick comment here might be helpful for future contributors in explaining why we have this branching here which we didn't have before